pyinterprod/pronto/signature.py

```diff
@@ -404,17 +409,17 @@ def insert_signatures(ora_uri: str, pg_uri: str, matches_file: str,
     logger.info("done")
 
 
-def _iter_comparisons(comps: dict[str, dict[str, list[int, int, int]]]):
+def _iter_comparisons(comps: dict[str, dict[str, list[int, int, int, int]]]):
     for acc1, others in comps.items():
-        for acc2, (collocations, protein_overlaps, _) in others.items():
-            yield acc1, acc2, collocations, protein_overlaps
-            yield acc2, acc1, collocations, protein_overlaps
+        for acc2, (collocations, protein_overlaps, _, rev_overlaps) in others.items():
+            yield acc1, acc2, collocations, protein_overlaps, rev_overlaps
+            yield acc2, acc1, collocations, protein_overlaps, rev_overlaps
```

Comment:
Renaming variables so they are shorter:

```suggestion
        for acc2, (collocs, prot_ovrs, _, rev_res_ovrs) in others.items():
            yield acc1, acc2, collocs, prot_ovrs, rev_res_ovrs
            yield acc2, acc1, collocs, prot_ovrs, rev_res_ovrs
```